strr-web/components/bcros/form-section/property/Form.vue

```diff
@@ -260,7 +260,8 @@ const validateHostResidence = () => {
 }
 
 const validateNumberOfRoomsForRent = () => {
-  if (!formState.propertyDetails.numberOfRoomsForRent || formState.propertyDetails.numberOfRoomsForRent <= 0) {
+  if (!formState.propertyDetails.numberOfRoomsForRent || formState.propertyDetails.numberOfRoomsForRent < 1) {
+    formState.propertyDetails.numberOfRoomsForRent = 1
```

Comment:
So you can decrease number of rooms to less than 1?

Reply:
no the user cannot decrement past 1

Reply:
Then we don't need this check and set to 1 ?